### ISSUE TITLE
Build failure in arm runner without any specific error

### ISSUE THREAD

**Issue by Biswa96** [msys2/msys2-autobuild#89]
1. kate 24.05.1
https://github.com/msys2-arm/msys2-autobuild/actions/runs/9519792468/job/26243761770#step:11:2005

This same issue happened before. @jeremyd2019

**Comment by jeremyd2019** [comment]
Yeah I remember this.  It also didn't happen when I built the package manually, or on my "ci" runner using MINGW-packages CI mechanism.  Very strange.

I did just make new runner instances for June's Patch Tuesday, and I also reduced the packages pre-installed (git zip and unzip).

**Comment by jeremyd2019** [comment]
It actually failed this time running manually.

**Comment by jeremyd2019** [comment]
Exit status -1073741819 - I think that's C0000005 - access violation?

**Comment by jeremyd2019** [comment]
I was trying to track it down and it stopped reproducing.  Restarted runner, see what happens

**Comment by jeremyd2019** [comment]
Well it worked that time.  Gave a bunch of module errors output.  I noticed it was failing to load those, but figured that was expected.  Now it's giving a bunch of output about them

**Comment by jeremyd2019** [comment]
I was thinking maybe defender was getting in the way, but changing its settings didn't help.  I noticed search indexer was using cpu, I was starting to mess with that when it started working again, maybe that was interfering?  I wonder if it would be safe to disable the windows search service as part of my runner setup, or if some package might try to talk to it

**Comment by jeremyd2019** [comment]
I've re-created the runners for with this month's updates.  I also manually switched WSearch service to demand start, so hopefully that will help.

**Comment by jeremyd2019** [comment]
I haven't seen any particular issues with WSearch service set to demand - I'm thinking about adding that to my configs for windows setup moving forward.  I haven't had to make a change to the basic windows setup (unattend.xml) in some time though.

**Comment by Biswa96** [comment]
kate failed to build again https://github.com/msys2-arm/msys2-autobuild/actions/runs/10555377688/job/29238890612#step:11:1243

**Comment by jeremyd2019** [comment]
I added setting wsearch service to demand (manual) start to a setup script this time, and I just confirmed that it is still demand start, but it was running.  I just stopped it and switched it to disabled.  I'm not even sure wsearch is the cause...

**Comment by Biswa96** [comment]
Could you upload the package manually?

**Comment by jeremyd2019** [comment]
Another month, another new runner instance - this time created with WSearch service disabled from the start, but I don't think that was actually the problem.

**Comment by Biswa96** [comment]
ark failed to build https://github.com/msys2-arm/msys2-autobuild/actions/runs/12725675540/job/35473187972?check_suite_focus=true#step:12:525

**Comment by jeremyd2019** [comment]
I ran it again with the new cmake update, but same result.  It seems to be doing better manually running on the autobuild runner.

**Comment by jeremyd2019** [comment]
Manually uploaded
